addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -702,6 +702,59 @@ export class OdooEditor extends EventTarget {
                 this.updateColorpickerLabels();
             });
         }
+        const fontSizeInput = this.toolbar.querySelector('input#fontSizeCurrentValue');
+        this.addDomListener(this.toolbar, 'click', ev => {
+            if (fontSizeInput && ev.target.closest('#font-size .dropdown-toggle')) {
+                // If the click opened the font size dropdown, select the input content.
+                fontSizeInput.select();
+            } else if (!this.isSelectionInEditable()) {
+                // Otherwise, if we lost the selection in the editable, restore it.
+                this.historyResetLatestComputedSelection(true);
+            }
+        })
+        // Handle the font size input.
+        if (fontSizeInput) {
+            const debouncedOnInputChange = (() => {
+                let handle;
+                return () => new Promise(resolve => {
+                    clearTimeout(handle);
+                    handle = setTimeout(() => {
+                        handle = null;
+                        setFontSize(fontSizeInput.value);
+                        resolve();
+                    }, 50);
+                });
+            })();
```

Comment:
Just re-read this quickly. Isn't there a `debounce` util somewhere already ?

Reply:
There is but importing it from /web breaks the tests (since those assets are not loaded in the tests) and I figured there'd be other cases than the tests where that would be problematic. So I stripped it down to its essence here, it's easier and safer I think.